src/region/regionUtil.ts

```diff
@@ -12,6 +12,29 @@ export const BORDER_SIZE = 4;
 export const BORDER_TOTAL = BORDER_SIZE * 2;
 export const EMPTY_STYLE = { display: 'none' };
 
+export const centerShape = (shape: Shape): Position => {
+    const { height, width } = shape;
+
+    return {
+        x: width / 2,
+        y: height / 2,
+    };
+};
+
+export const centerRegion = (shape: Shape): Position => {
+    const { x: shapeX, y: shapeY } = shape;
+    const { x: centerX, y: centerY } = centerShape(shape);
+
+    return {
+        x: centerX + shapeX,
+        y: centerY + shapeY,
+    };
+};
```

Comment:
Should we combine these two functions into one? `centerShape` is only used once and its logic is pretty simple.

Reply:
I have to disable prettier for a couple lines if I combine them due to poor parenthesis handling in Prettier@1.x.